src/realm/object-store/schema.hpp

```diff
@@ -54,6 +55,10 @@ class Schema : private std::vector<ObjectSchema> {
     iterator find(ObjectSchema const& object) noexcept;
     const_iterator find(ObjectSchema const& object) const noexcept;
 
+    // find an ObjectSchema by table key
+    iterator find(TableKey table_key) noexcept;
```

Comment:
Seems to be missing tests for both of these?

Reply:
On the .NET team we have a policy not to write bugs, which dramatically simplifies testing 😜

Fair point though - @papafe do you want to try your hand at some C++ tests or would you prefer to bribe @fealebenpae with some bonusly to have someone on the Core team take over.

Reply:
Mmmh, I've given a look at how the tests are written... and I think that bribing is the best option to do this somehow fast   :grin: